### PR TITLE
[web] Librarify all remaining files

### PR DESCRIPTION
**Pull request by mdebbar**
Librarify the last batch of part files.

### CODE REVIEW

File: lib/web_ui/lib/src/engine/window.dart [hunk 1]
```diff
@@ -2,12 +2,30 @@
...
+library window;
+
+import 'dart:async';
+import 'dart:html' as html;
+import 'dart:typed_data';
+
+import 'package:js/js.dart';
+import 'package:meta/meta.dart';
+import 'package:ui/src/engine.dart' show registerHotRestartListener;
+import 'package:ui/ui.dart' as ui;
```
Comment (yjbanov) [inline lib/web_ui/lib/src/engine/window.dart:15]: We should probably be consistent and either use relative imports or `package:` imports, but not both.

**Review by yjbanov — APPROVED** [review]
lgtm

**Comment by ditman** [comment]
Did this cause an infinite loop or something?

```
00:20 [32m+46[0m[33m ~2[0m[31m -2[0m: test/engine/surface/frame_timings_test.dart: collects frame timings [1m[31m[E][0m[0m                                                                                                               
   Stack Overflow
   ../../../../lib/src/engine/platform_dispatcher.dart 143:41  onBeginFrame
   ../../../../lib/src/engine/platform_dispatcher.dart 143:41  onBeginFrame
   ../../../../lib/src/engine/platform_dispatcher.dart 143:41  onBeginFrame
  ...
```

File: lib/web_ui/lib/src/engine/platform_dispatcher.dart [hunk 2]
```diff
@@ -115,7 +140,7 @@ class EnginePlatformDispatcher extends ui.PlatformDispatcher {
   /// [PlatformWindow.scheduleFrame] has been called since the last time this
   /// callback was invoked.
   @override
-  ui.FrameCallback? get onBeginFrame => _onBeginFrame;
+  ui.FrameCallback? get onBeginFrame => onBeginFrame;
```
Comment (mdebbar) [inline lib/web_ui/lib/src/engine/platform_dispatcher.dart:143]: @ditman  haha of course this is an infinite loop.

File: lib/web_ui/lib/src/engine/window.dart (hunk 1 above)
Comment (mdebbar) [reply lib/web_ui/lib/src/engine/window.dart:15]: I'm already preparing another PR that does final cleanup. I'll change to relative paths there.

File: lib/web_ui/lib/src/engine/platform_dispatcher.dart (hunk 2 above)
Comment (ditman) [reply lib/web_ui/lib/src/engine/platform_dispatcher.dart]: Only Chuck Norris would have been able to finish executing `onBeginFrame`! 😭

**Pull request merged**